CHANGELOG.md

```diff
@@ -20,6 +20,12 @@ All Notable changes to `Backpack CRUD` will be documented in this file
 - Nothing
 
 
+## [3.2.20] - 2017-09-x
+
+## Fixed
+- zh-cn instead of zh-CN language folder - fixes #849;
```

Comment:
https://github.com/Laravel-Backpack/CRUD/issues/849 is fixed on dev; apparently.

Reply:
Merging it there now.